test/src/factories/checkFactory.ts

```diff
@@ -12,7 +12,7 @@ const checkFactory = (
   keys: IKeyList,
 ): void => {
   describe.each([...keys])(`When ${factoryName} is called with %s`, (key) => {
-    const validatedKey = key.replaceAll('.', '_').replaceAll('/', '__');
```

Comment:
`.replaceAll` is added in node15. This is not supported in the current GitHub action or GCP server.